Adding tko-vwt-dg 2.0 (#194)
vmware-tanzu-labs/tanzu-validated-solutions: src/deployment-guides/tko-on-vsphere-with-tanzu.md

After the controller VM is deployed and powered-on, configure the controller VM for your vSphere with Tanzu environment. The controller requires several post-deployment configuration parameters.

On a browser, go to https://<https://<alb-ctlr01.tanzu.lab>/.

1. Configure an **Administrator Account** by setting up a password and optionally, an email address.

  ![Screenshot of Create Account screen](img/tko-on-vsphere-with-tanzu/TKO-VWT02.png)

2. Configure **System Settings** by specifying the backup passphrase and DNS information.

  ![Screenshot of System Settings screen](img/tko-on-vsphere-with-tanzu/TKO-VWT03.png)

3. (Optional) Configure **Email/SMTP**.

  ![Screenshot of Email/SMTP screen](img/tko-on-vsphere-with-tanzu/TKO-VWT04.png)

4. Configure **Multi-Tenant** settings as follows:

   - IP Route Domain: Share IP route domain across tenants.
   - Service Engine Context: Service Engines are managed within the tenant context, not shared across tenants.

   ![Screenshot of Multi-Tenant screen](img/tko-on-vsphere-with-tanzu/TKO-VWT05.png)

5. Click on **Save** to finish the post-deployment configuration wizard.

>    If you did not select the **Setup Cloud After** option <_I don't see this option_> before saving, the initial configuration wizard exits. The Cloud configuration window does not automatically launch and you are directed to a Dashboard view on the controller.

Reviewer: ```suggestion
   If you did not select the **Setup Cloud After** option before saving, the initial configuration wizard ends. The cloud configuration window does not automatically launch and you are directed to a dashboard view on the controller.
```

Reviewer: I removed my embedded comment **<_I don't see that option_>** from the MD file so I could explain it here, which is a better place for comments. When I reviewed this guide, I didn't see where the **Setup Cloud After** option is.  Was it shown in one of the screenshots you replaced, or did I just miss it? Please verify that the option is still in the wizard, and if it is, consider rewording this line so it's clearer.
